src/packages/shared-types/action-types/withdrawPackage.ts

```diff
@@ -0,0 +1,21 @@
+import { z, ZodArray } from "zod";
+import { onemacAttachmentSchema } from "../onemac";
+
+export const withdrawPackageSchema = (attachmentArrayType: ZodArray<any>) =>
```

Comment:
Attachment arrays are two different things to front and back end: front expects a `File` when it validates form data, but the back-end events are expecting the `onemacAttachmentSchema` type for array items. This way, we can use a single Zod validation model with just a small variance in the expected data in the attachment array(s)